internal/quickstart/toggle_flag.go

```diff
@@ -1,18 +1,30 @@
 package quickstart
 
 import (
+	"fmt"
 	"github.com/charmbracelet/bubbles/key"
 	tea "github.com/charmbracelet/bubbletea"
 	"github.com/charmbracelet/lipgloss"
+	"ldcli/internal/flags"
 )
 
 type toggleFlagModel struct {
-	flagKey string
+	accessToken string
+	baseUri     string
+	client      flags.Client
+	enabled     bool // whether the flag has ever been toggled on
```

Comment:
Could you rename this something like `flagWasToggled` since I think that `enabled` means the flag is currently toggled on and not that it was changed at all.